src/Microsoft.ML/TextLoader.cs

```diff
@@ -1,125 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using Microsoft.ML.Runtime;
-using Microsoft.ML.Runtime.Api;
-using Microsoft.ML.Runtime.Data;
-using Microsoft.ML.Runtime.EntryPoints;
-using System;
-using System.Linq;
-using System.Reflection;
-using System.Text;
-
-namespace Microsoft.ML
-{
-    public class TextLoader<TInput> : ILearningPipelineLoader
-    {
-        private string _inputFilePath;
-        private string CustomSchema;
-        private Data.TextLoader ImportTextInput;
-
```

Comment:
I'm not a fan of this old code, but it had the benefit that `TInput` meant something. Now we're doing inheritance, it means nothing. If you want to generate a `TextLoader`, make a method to generate a text loader.